client.go

```diff
@@ -0,0 +1,280 @@
+package globalsign
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"sync"
+	"time"
+)
+
+// Client globasign client.
```

Comment:
// Client implements a client for Globalsign Atlas DSS API service.

Reply:
Updated at 6fa006e290519f3ae08c512c53671a7b8ce4e71f